blueprints/cloud-operations/network-dashboard/deploy-cloud-function/outputs.tf

```diff
@@ -16,12 +16,7 @@
 
 output "bucket" {
   description = "Cloud Function deployment bucket resource."
-  value       = module.cloud-function.bucket
-}
-
-output "cloud-function" {
-  description = "Cloud Function resource."
-  value       = module.cloud-function.function
+  value       = var.cloud_function_config.version == "v1" ? module.cloud-function.0.bucket : module.cloud-function-v2.0.bucket
```

Comment:
I'd create a local like this:
```terraform
locals {
  function = (
    var.cloud_function_config.version == "v1" 
    ? module.cloud-function.0 
    : module.cloud-function-v2.0
  )
}
```

With this you can just do `value = local.function.bucket`

Reply:
Good idea, thanks!